packages/slate-react/src/plugins/before.js

```diff
@@ -32,6 +32,17 @@ function BeforePlugin() {
   let isCopying = false
   let isDragging = false
 
+
+  /**
+   * The before plugin queries.
+   *
+   * @type {Object}
+   */
+
+  const queries = {
+    isComposing: () => isComposing,
```

Comment:
This is genius @yunwuxin! I never thought about exposing this information via queries like this, cool idea.